maquetado/index.html

```diff
@@ -6,9 +6,22 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+	<link rel="stylesheet"  type="text/css" href="application.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
 	<h2 class="title">Front-End Tools</h2>
+	<div class="technology">
+		<div class="border-ext">
+			<div class="border-int">
+				<img class="border-ext-img" src="./assets/react_icon.svg">
+				<h4 class="border-ext-h4">ReactJS</h4>
```

Comment:
I dont think both of this ones are `h4`, one is a title and the other one a subtitle, `h5` maybe?

Also, class names are not very descriptive, what about `tech-title` and `tech-release`?